src/components/ebay-menu-button/examples/19-prefix-label/template.marko

```diff
@@ -0,0 +1,8 @@
+<ebay-menu-button type="radio" prefix-label="Select From Menu">
+    <@label>
+        label text
+    </@label>
+    <@item checked>item 1 that has very long text</@item>
```

Comment:
Let's remove this very long text option in both examples and use something more real-life for each option.